localstack/services/ses/ses_starter.py

```diff
@@ -128,6 +131,32 @@ def get_identity_verification_attributes(self):
 
     email_responses.get_identity_verification_attributes = get_identity_verification_attributes
 
+    backend_send_email_orig = SESBackend.send_email
+
+    def send_email_save_contents(self, source, subject, body, destinations, region):
+        message = backend_send_email_orig(self, source, subject, body, destinations, region)
+
+        data_dir = config.DATA_DIR
+        if data_dir == '':
+            data_dir = config.TMP_FOLDER
+
+        ses_dir = os.path.join(data_dir, 'ses')
+
+        if not os.path.exists(ses_dir):
+            os.makedirs(os.path.join(data_dir, 'ses'))
+        f = open(os.path.join(ses_dir, message.id + '.json'), 'w')
```

Comment:
nitpick: we usually try to avoid `.close()`, and use `with` blocks, to ensure file descriptors are closed and not leaked (in case of errors):
```
with open(os.path.join(ses_dir, message.id + '.json'), 'w') as f:
    f.write(json.dumps({ ...
```

Reply:
Will fix that! :)